amis/packer_centos8.json

```diff
@@ -0,0 +1,308 @@
+        "curl --retry 3 -L ${chef_install_url} | sudo bash -s -- -v {{user `chef_version`}}"
+      ]
+    },
+    {
+      "type" : "shell",
+      "inline" : [
+        "sudo /opt/cinc/embedded/bin/gem install --no-document berkshelf:{{user `berkshelf_version`}}"
+      ]
+    },
+    {
+      "type" : "shell",
+      "inline" : [
+        "set -x",
+        "sudo /opt/cinc/embedded/bin/berks vendor /etc/chef/cookbooks --berksfile /tmp/cookbook/Berksfile",
+        "sudo chown -R root:root /etc/chef"
+      ]
+    },
+    {
+      "type" : "chef-solo",
+      "only": ["centos8"],
+      "remote_cookbook_paths" : [
+        "/etc/chef/cookbooks"
+      ],
+      "skip_install" : "true",
+      "execute_command" : "sudo chef-client -z --no-color -c {{.ConfigPath}} -j {{.JsonPath}}",
+      "json" : {
+        "cfncluster" : {
+          "default_pre_reboot" : "false"
+        }
+      },
+      "run_list" : [
+        "aws-parallelcluster::_default_pre"
+      ]
+    },
+    {
+      "type" : "shell",
+      "only": ["centos8"],
+      "expect_disconnect" : "true",
+      "inline" : [
+        "sudo reboot"
+      ]
+    },
+    {
+      "type" : "chef-solo",
+      "only": ["centos8"],
+      "pause_before": "2m",
+      "json" : {
+        "cfncluster" : {
+          "cfn_region": "{{user `region`}}",
+          "nvidia" : {
+            "enabled" : "{{user `nvidia_enabled`}}"
+          },
+          "dcv": {
+            "installed" : "{{user `dcv_installed`}}"
+          },
+          "custom_node_package" : "{{user `custom_node_package`}}",
+          "cfn_base_os" : "centos8"
+        }
+      },
+      "remote_cookbook_paths" : [
+        "/etc/chef/cookbooks"
+      ],
+      "skip_install" : "true",
+      "execute_command" : "sudo chef-client -z --no-color -c {{.ConfigPath}} -j {{.JsonPath}}",
+      "run_list" : [
+        "aws-parallelcluster::default"
+      ]
+    },
+    {
+      "type" : "chef-solo",
+      "only": ["custom-centos8"],
+      "remote_cookbook_paths" : [
+        "/etc/chef/cookbooks"
+      ],
+      "json" : {
+        "cfncluster" : {
+          "cfn_base_os" : "centos8"
+        }
+      },
+      "skip_install" : "true",
+      "execute_command" : "sudo chef-client -z --no-color -c {{.ConfigPath}} -j {{.JsonPath}}",
+      "run_list" : [
+        "aws-parallelcluster::default"
+      ]
+    },
+    {
+      "type" : "shell",
+      "inline" : [
+        "region=\"{{user `region`}}\"",
+        "bucket=\"s3.amazonaws.com\"",
+        "[[ ${region} =~ ^cn- ]] && bucket=\"s3.cn-north-1.amazonaws.com.cn/cn-north-1-aws-parallelcluster\"",
+        "curl --retry 3 -L -o /tmp/aws-cfn-bootstrap-latest.tar.gz https://${bucket}/cloudformation-examples/aws-cfn-bootstrap-latest.tar.gz",
+        "which pip2",
+        "if [ $? -eq 0 ]; then sudo pip2 install /tmp/aws-cfn-bootstrap-latest.tar.gz; else sudo pip install /tmp/aws-cfn-bootstrap-latest.tar.gz; fi"
```

Comment:
Probably we'll need to add symlinks for `cfn-init` and `cfn-signal` scripts:
https://computingforgeeks.com/how-to-install-aws-cloudformation-helper-scripts-on-centos/